### PR TITLE
Gavilya version v.4.3.0.2312

### PR DESCRIPTION
**Pull request by Bamboooz**
Gavilya version v.4.2.0.2312:

CHANGELOG:
- Added Gavilya icon in the top left side of the app
- Adjusted settings button margin which was slightly off
- When hovering on games the text was "Play to {game_name}" instead of "Play {game_name}"
- Changed the blue gradient in the displayed images when nothing is in the library/favorites page to a purple gradient to fit the applications style
- Changed the play button gradient from blue to purple, and in the home page display the play button only when hovering on game card, instead of it always being there
- Slight margin changes in the game card view
- Steam games are now launched using steam://rungameid/{game_id} which launches the steam with it if it is not already opened,
which means you don't have to have to open steam yourself.
- Created a separate favorites page, instead of favorited games being in the navbar. Favorite games tab has been removed from the library list view and you can see the favorited games normally in
 the games tab, they just have a star icon on them.
- Fixed many code style inconsistencies, like double free lines somewhere, in one files no space between the license info and in others there was one etc.
- Added a python script to add license info to all C# files if it is not already in them
- "What game do you wanna play" text in the home page replaced with "What game do you want to play"
- Added a "this week" tab to the recent page
 
 obraz_2023-12-05_163446827
 
image

image

image

image

### CODE REVIEW

**Comment by Bamboooz** [comment]
No problem, very fun project to work on, there is just one bug left to fix - I've been working on it for a few days, but I can't find the cause. When in the settings you select favorites page as the main page, the navbar button highlight on the favorites page button does not appear, but it does when you open that page on your own.

**Comment by Bamboooz** [comment]
And maybe other small things like some README changes, like the version text change in it, as I put this pull request as as a whole update as there is quite a lot of changes there. When I will be home today I will check for any inconsistencies and address them in an additional commit.

File: Gavilya/Properties/Resources.fr-FR.resx [hunk 1]
```diff
@@ -1023,4 +1023,16 @@
...
     <value>Scanner un dossier</value>
   </data>
+  <data name="SteamAppNotInstalled" xml:space="preserve">
+    <value>Impossible d'exécuter ce jeu Steam car il n'est pas installé.</value>
+  </data>
+  <data name="AddFavGameWelcome" xml:space="preserve">
+    <value>Ajoutez un jeu favori en cliquant sur le bouton "Étoile" sur la carte d'un jeu.</value>
+  </data>
+  <data name="AddFavGame" xml:space="preserve">
+    <value>Ajouter un jeu favori</value>
```
Comment (Leo-Peyronnet) [inline Gavilya/Properties/Resources.fr-FR.resx:1033]:
```suggestion
    <value>Ajouter un jeu aux favoris</value>
```

**Comment by Bamboooz** [comment]
Give me 1 more day, I am going to do some finishing touches.

**Review by Leo-Peyronnet — APPROVED** [review]

**Comment by Leo-Peyronnet** [comment]
Thank you again for your contribution!

**Pull request merged**